src/ui/qgstextformatwidgetbase.ui

```diff
@@ -4547,22 +4548,61 @@ font-style: italic;</string>
                           <number>0</number>
                          </property>
                          <item row="0" column="0">
-                          <widget class="QLabel" name="mCentroidLabel">
+                          <widget class="QLabel" name="label_14">
```

Comment:
Remove that label and just have the checkbox speak for itself. It'll help avoid creating to large a minimum width situation.